.gitignore

```diff
@@ -2,3 +2,10 @@ vendor/
 cmd/commento/commento
 *.db
 .idea
+.env
```

Comment:
@DarkThrone mentioned having a default environment that developers can use out of the box  so there is one environment file that is not gitignored and that is .env.development.
Because I allow one of the 8 or 9 environments to be checked in, I had to specify the others individually.